src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -125,6 +125,58 @@ const AccessibilityRequestDetailPage = () => {
 
   const userGroups = useSelector((state: AppState) => state.auth.groups);
   const isAccessibilityTeam = user.isAccessibilityTeam(userGroups, flags);
+  const hasDocuments = documents.length > 0;
+
+  const uploadDocumentLink = (
```

Comment:
I think it's best practice to capitalize these local vars since they're JSX objects?

Reply:
I'm bad with words so I'll talk with code.

```
1. 
const someJSX = (
   <div>
     <p>Hello</p>
  </div>
)

Usage:
return (
  <section>
    {someJSX)
  </section>
)

2.
const SomeJSX = () => (
   <div>
     <p>Hello</p>
  </div>
)

Usage:
return (
  <section>
    <SomeJSX />
  </section>
)
```

I think both are valid. I would prefer not doing `2` within a functional component. We've had some nasty bugs with that. IMO, `1` is fine. `2` is also fine, just declare the component on its own and not inside another component.

I think I'm in the minority, but I prefer everything in the `return` in functional components (with a few exceptions). I can talk about that another time.

Reply:
I changed and merged before I saw this comment.  I definitely did not do what you recommended regarding declaring the component on its own.  If we go with your recommendations, @christopherhuii , I I'd change it back to what I had before since that code is dependent on values received by the component.